src/transformers/generation/configuration_utils.py

```diff
@@ -183,7 +183,7 @@ class GenerationConfig(PushToHubMixin):
             The id of the *padding* token.
         bos_token_id (`int`, *optional*):
             The id of the *beginning-of-sequence* token.
-        eos_token_id (`int`, *optional*):
+        eos_token_id (`Union[int, List[int]]`, *optional*):
             The id of the *end-of-sequence* token.
```

Comment:
Can we adapt the text of the doc here? The docstrings also need to be updated in `beam_search.py` FYI.